### ISSUE TITLE
webpack build broken (Cannot resolve module 'fs')

### ISSUE THREAD

**Issue by froatsnook** [kjur/jsrsasign#182]
From my early tests, jsrsasign is *almost* working with webpack.  The issue I found is that the functions in `nodeutil-1.0.js` call `require("fs")` which results in the error `Module not found: Error: Cannot resolve module 'fs' in ./node_modules/jsrsasign/lib`.

The workaround is to put the following in `webpack.config.js`:

```javascript
module.exports = {
  ...
  node: {
    fs: "empty",
  },
};
```

In my opinion, these util functions don't really belong in this library!  Would you accept a pull request that removed `nodeutil-1.0.js` functions from the npm build?

**Comment by jmarceli** [comment]
I've got similar issue. It seems that this is recent versions bug, see: https://github.com/kjur/jsrsasign/issues/143.

The workaround is to downgrade package to 5.0.1 version.

**Comment by froatsnook** [comment]
Yes, this is definitely a duplicate of #143.  Thanks for the suggestion :).

**Comment by kjur** [comment]
I've released 6.0.0 today which is now Node.js and "fs" safe. Please check it.
